zebra-rpc/src/methods/tests/snapshot/snapshots/get_block_subsidy_future_nu6_height@testnet_10.snap

```diff
@@ -0,0 +1,26 @@
+---
+source: zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs
+expression: get_block_subsidy
+---
+{
+  "lockboxstreams": [
+    {
+      "recipient": "Lockbox",
+      "specification": "https://zips.z.cash/zip-0214",
+      "value": 0.1875,
+      "valueZat": 18750000
+    },
+    {
+      "recipient": "Major Grants",
```

Comment:
One of the things that is not clear to me and that we need to match `zcashd` is this.

- The final decision about the dev fund is part lockbox - part FPF.
- Zebra implements the lockbox just as a new funding stream.
- In the RPC method we change the array name to `lockboxstreams` if we are after NU6.
- Does the major grant recepient belongs to the lockbox streams or should be part of the funding streams array ?
- Should we change `"Major Grants` to `FPF` after NU6 activation ?

Reply:
> The final decision about the dev fund is part lockbox - part FPF.

[Alternative 2 here](https://zips.z.cash/draft-nuttycom-funding-allocation) won the run-off, so it'll be part FPF (8%) and part lockbox (12%) for 2 years.

> Zebra implements the lockbox just as a new funding stream.

Yep, but one without an address, so in `miner_subsidy()`, the lockbox stream is still subtracted from the block subsidy, but it's not added as a funding stream output to the coinbase transaction template.

> In the RPC method we change the array name to `lockboxstreams` if we are after NU6.

There's no change to the `fundingstreams` field except that it'll be excluded entirely for block heights that don't have any funding streams.

The `lockboxstreams` field is being added for block heights in the post-NU6 funding streams height range, it'll have only 1 item in there for now.

> Does the major grant recepient belongs to the lockbox streams or should be part of the funding streams array ?

It should be added to the `fundingstreams` array.

> Should we change `"Major Grants` to `FPF` after NU6 activation ?

I don't see any changes in zcashd around the name or anything to suggest a name change in the ZIPs so I think we should leave it for now.